eng/pipelines/templates/stages/archetype-java-release.yml

```diff
@@ -63,7 +63,7 @@ stages:
             steps:
               - task: PowerShell@2
                 displayName: Prep template pipeline for release
-                condition: and(succeeded(),eq(variables['TestPipeline'],'true'))
+                condition: and(succeeded(),eq('${{ parameters.TestPipeline }}','true'))
```

Comment:
Since we are using template parameters I think it would be good to use a template condition so most pipelines don't see this step in their pipelines.